### PR TITLE
test(quench): Introduce Quench tests for integration testing

### PR DESCRIPTION
**Pull request by bheiskell**
I wrote a sample test for the datapads features. I'd say this style of testing is effective.

Quench has easier access to the actual FoundryVTT API than Cypress does. Technically, I think these tests could be written with Cypress, but I think this approach is less cumbersome for development. It should be reasonably straightforward to convert these to Quench tests if we decide we want to do that later.

Unrelated to Quench, I noticed finding the right FoundryVTT hook did prove challenging at times. A lot of interactions with things like Dialogs don't really expose an "I'm done" hook. We may need to introduce hooks in our application at points to communicate, "I'm done". These aren't ideal, but I suspect pragmatism will win out in certain circumstances.

Closes: #144

### CODE REVIEW

**Comment by bheiskell** [comment]
I think the next step should be splitting up the cypress tests into multiple files, but I didn't want to put all that into this change and make it hard to follow what actually changed in `home.cy.js`.

**Comment by bheiskell** [comment]
@wrycu My suggestion is to review this, then https://github.com/bheiskell/StarWarsFFG-Enhancements/pull/19 . I can pull those commits into this PR if you're happy with them. I just wanted to keep them separate for ease of initial reviewing.

**Review by wrycu — APPROVED** [review]
LGTM
